css/fieldmanager-group-tabs.css

```diff
@@ -89,13 +118,27 @@
 
 .fm-wrapper .wp-tabs-panel > .fm-wrapper .fm-group-inner {
 	border-top: solid 1px #dfdfdf;
+	border-right: none;
+	border-bottom: none;
+	border-left: none;
+	-webkit-box-shadow: none;
+	box-shadow: none;
 }
 
 /* Vertical Tabs */
-.fm-tabbed-vertical {
-	margin: -6px -12px -12px;
-	background: url(../images/vertical-tab-bg.png) 0 0 repeat-y;
+#fm_meta_box_vertical_tabbed_meta_fields .inside {
+	margin-top: 0;
+	padding: 0;
 }
+
+#fm_meta_box_vertical_tabbed_meta_fields .fm-group-description {
+	padding: 0 12px 12px;
+}
+
+#fm_meta_box_vertical_tabbed_meta_fields .wp-tabs-panel > .fm-wrapper .fm-group-inner {
```

Comment:
I see the original selector was `.fm-tabbed-vertical` – any particular reason that would no longer work? I'd try to avoid ID selectors if possible, as I would think there could be more than one of these metaboxes per page?.

Reply:
yeah, this predates my memory. I agree.